### PR TITLE
this need for android, in deep sleep android sometimes ignore RA.

### PR DESCRIPTION
**Pull request by 143981**
why maximum 9000 for 2 bytes? android needs maximum because his lost default router with 9000.

### CODE REVIEW

**Comment by 143981** [comment]
https://datatracker.ietf.org/doc/rfc8319/

>   This document updates Sections 4.2 and 6.2.1 of [RFC4861] to change
>    the following router configuration variables.
> 
>    In Section 4.2, inside the paragraph that defines Router Lifetime,
>    change 9000 to 65535 seconds.
> 
>    In Section 6.2.1, inside the paragraph that defines
>    MaxRtrAdvInterval, change 1800 to 65535 seconds.
> 
>    In Section 6.2.1, inside the paragraph that defines
>    AdvDefaultLifetime, change 9000 to 65535 seconds.

**Comment by stappersg** [comment]
It will need a proper commit message.

**Comment by stappersg** [comment]
It still misses a proper commit message.

For examples of good commit messages:
```text
cd  .../directory/of/git/clone/of/linux/source
git log --author 'Geert*'
```

No hard feelings, just resubmit (with a nice commit message).

**Comment by 143981** [comment]
I don't know English that well, sorry, i can't do good commit, but this change need for android...

**Comment by stappersg** [comment]
On Fri, Sep 30, 2022 at 08:45:14AM -0700, 143981 wrote:
> I don't know English that well, sorry, i can't do good commit, but this change need for android...

No hard feelings, just resubmit (with a nice commit message).


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by 143981** [comment]
i try commit to my branch

**Comment by Neustradamus** [comment]
@143981: Have you done it?

**Comment by 143981** [comment]
i don't understand how i can push into closed pr...

**Comment by Neustradamus** [comment]
@143981: You can do a new PR with a good commit...

**Comment by 143981** [comment]
https://github.com/radvd-project/radvd/pull/192

**Comment by Neustradamus** [comment]
Replaced by @Leseratte10 PR:
- https://github.com/radvd-project/radvd/pull/265
